x/shield/legacy/v3/store_test.go

```diff
@@ -0,0 +1,167 @@
+package v3
+
+import (
+	v2 "github.com/certikfoundation/shentu/v2/x/shield/legacy/v2"
+	"github.com/certikfoundation/shentu/v2/x/shield/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"testing"
+	"time"
+)
+
+const (
+	DENOM           = "CTK"
```

Comment:
probably should use the default bond denom `sdk.DefaultBondDenom` or `uctk` from `common` package

Reply:
sure, will make the change.